.github/workflows/ci.yml

```diff
@@ -2,8 +2,6 @@ name: ci
 
 on:
   push:
-    branches:
-      - master
```

Comment:
What would this achieve? (My knowledge of GitHub actions is limited.) I found this documentation:
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration
If I was to guess... The intention was to ensure that Travis / AppVeyor tests were triggered for external contribution pull requests like #2156. Also, I cannot even find the file `.github/workflows/ci.yml` in the repo: https://github.com/balena-io/balena-cli/tree/master/.github
But this PR clearly amends an existing file rather than add a new file, which makes me properly confused. :-)